plugin.video.media-ccc-de/addon.xml

```diff
@@ -1,8 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.media-ccc-de" name="media.ccc.de" version="0.1.0" provider-name="Tobias Gruetzmacher">
+<addon id="plugin.video.media-ccc-de" name="media.ccc.de" version="0.1.1" provider-name="Tobias Gruetzmacher">
   <requires>
     <import addon="xbmc.python" version="2.24.0"/>
-    <import addon="script.module.simplejson" version="3.3.0"/>
```

Comment:
why did you remove simplejson? note that your addon is in the jarvis branch which means you cannot effectively rely on the json module in every kodi platform. Or does it only affect unit tests?

Reply:
you can certainly rely on json as it was in python2.5 and up. It just might be a bit slower

Reply:
I don't think the performance difference between simplejson and json is such a big deal and shouldn't everyone be on Python 2.7 nowadays (yeah, I know: hopes and dreams ;))?

Anyways, I was only using simplejson for tests and I rely on whatever requests is using (which might be simplejson AFAICS, but it also runs without) when parsing the "real" data.

Related: Are there statistics which Kodi versions are still in use? (e.g. addon repository meta data downloads split by user agent?)

Reply:
3/4 is Krypton or up